summariser/README.md

```diff
@@ -0,0 +1,50 @@
+# Wind Tunnel: Summariser
+
+This is a tool to summarise the results of Wind Tunnel scenario runs.
+
+Start by running one or more scenarios with `--reporter influx-file` to create metrics on InfluxDB. You should find a
+`run_summary.jsonl` file created in the root of the project. The summariser reads this to find metrics.
+
+You can then run the summariser to generate a report:
+
+```shell
+RUST_LOG=info cargo run summariser 
+```
+
+This will create a new JSON file that summarises the results from the scenarios.
```

Comment:
Should this say `JSONL` everywhere where summaries are generated? It won't be valid JSON, I believe.

Reply:
The final report is valid JSON, I'm serializing a `Vec` of reports so the file will contain an array at the top level. It's entirely possible I've mixed up my file extensions in a few places, but I don't think here